src/appwindow.hpp

```diff
@@ -57,6 +57,8 @@ class AppWindow : public QMainWindow
 {
     Q_OBJECT
 
+    friend class Application;
```

Comment:
Dont use friend class. This will break the oops principles. Use setter getter if you want any attribute access from the class